frontends/web/src/routes/account/add/add.tsx

```diff
@@ -58,46 +60,35 @@ class ManageAccount extends Component<Props, State> {
     public componentDidMount() {
         backendAPI.getSupportedCoins()
             // TEST with only 1 coin
-            // .then(() => (['btc']))
+            // .then(() => ([{coinCode: 'tbtc' as accountApi.CoinCode, name: 'Bitcoin Testnet', canAddAccount: true}]))
             .then((coins) => {
                 const onlyOneSupportedCoin = (coins.length === 1);
                 this.setState({
-                    // @ts-ignore
-                    coinCode: onlyOneSupportedCoin ? coins[0] : 'choose',
-                    supportedCoins: coins
+                    coinCode: onlyOneSupportedCoin ? coins[0].coinCode : 'choose',
+                    step: onlyOneSupportedCoin ? 'choose-name' : 'select-coin',
```

Comment:
what if `coin.canAddAccount` is false but `onlyOneSupportedCoin` is true, e.g. you try to add account when you already have 5 accounts with btc-only?

Reply:
Yes good point, currently the dropdown options are grayed out, without a message and it wont let the user select one.

We need a guide entry for sure, and maybe even a fullscreen message "sorry you used all your accounts" or similar

Reply:
I think most clear way would be to not to grey out options, but to let the user select whichever coin they want. And if the coin the selected already has max. accounts, then a message is displayed saying they can't make more accounts. Something like "You cannot make any more [coin] accounts. Maximum number in 5."

Of course we can make it a user guide entry, and maybe that is enough for MVP.

Reply:
Would be nice if max-5-accounts it was communicated earlier, before any action